### DISCUSSION TITLE
Suggesting edits

### DISCUSSION THREAD

**Discussion by Joseph-Edwards** [tdhc153/starmast#21] [General]
I noticed some typos in one of the files (see issue #19) but wasn't sure of the best way to correct them. My options were to edit the master branch directly, or create a new branch and then open a pull request. I opted for the latter (see pull request #20) as this felt like the safest way, but slightly more effort.

Does anyone have any thoughts on this? Does your opinion change if the suggested edit is more significant than a typo?

**Comment by tdhc153** [comment]
Typos could be corrected by branch owners, but I was happy with the way this operated. Safety is best!
